notebooks/MUS2019_CTF.ipynb

```diff
@@ -0,0 +1,1120 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "MUS2019-CTF.ipynb",
+      "version": "0.3.2",
+      "provenance": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python2",
+      "display_name": "Python 2"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/obsidianforensics/timesketch/blob/ctf-colab/notebooks/MUS2019_CTF.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "2y1Dij2Z7C4n",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# Magnet User Summit CTF 2019\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "V3DWqc_275Jm",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The folks at [Magnet Forensics](https://www.magnetforensics.com/) had a [conference](https://magnetusersummit.com/) recently, and as part of it they put together a digital forensics-themed Capture the Flag competition. I wasn't able to attend, but thankfully they [released the CTF online](https://www.hecfblog.com/2019/04/daily-blog-657-mus2019-dfir-ctf-open-to.html) a few days after the live competition ended. \n",
+        "\n",
+        "It looked like a lot of fun and I wanted to take a crack at it using the open source tools we use/build here at Google. \n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "1_pWpmTdnYJM",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Forensics Preprocessing"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "2Hph_KETIvOT",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "I'm going to focus on how to find the answers to the CTF questions after all the processing has been done. I'll quickly summarize the processing steps I did to get to the state when I pick up my walkthrough.\n",
+        "\n",
+        "I started off by processing the provided E01 image with a basic log2timeline command; nothing special added:\n",
+        "\n",
+        "\n",
+        "```\n",
+        "ryan:~$ log2timeline.py MUS2019-CTF.plaso MUS-CTF-19-DESKTOP-001.E01\n",
```

Comment:
I would remove this "ryan" and just have

$ log2....

Reply:
Style choice, but done.